vite.config.js

```
@@ -1,52 +1,114 @@
-        }),
-    ],
-    define: {
-        '__APP_VERSION__': JSON.stringify(pkg.version),
+          },
+        ],
+      },
+    }),
+  ],
+  resolve: {
+    alias: {
+      "@": path.resolve(__dirname, "src"),
```

ESLint

[error] '__dirname' is not defined.

(no-undef)